packages/experiments/src/components/Button/Button.types.tsx

```diff
@@ -80,49 +80,49 @@ export interface IButtonProps
 export interface IButtonViewProps extends IButtonProps {}
 
 export interface IButtonTokens {
+  backgroundClip?: IRawStyleBase['backgroundClip'];
   backgroundColor?: string;
   backgroundColorHovered?: string;
   backgroundColorPressed?: string;
-  highContrastBackgroundColor?: string;
-  highContrastBackgroundColorHovered?: string;
-  highContrastBackgroundColorPressed?: string;
-  color?: string;
-  colorHovered?: string;
-  colorPressed?: string;
-  highContrastColor?: string;
-  highContrastColorHovered?: string;
-  highContrastColorPressed?: string;
   borderColor?: string;
   borderColorFocused?: string;
   borderColorHovered?: string;
   borderColorPressed?: string;
+  borderRadius?: number | string;
+  borderWidth?: number | string;
+  color?: string;
+  colorHovered?: string;
+  colorPressed?: string;
+  contentPadding?: number | string;
+  contentPaddingFocused?: number | string;
+  cursor?: string | undefined;
```

Comment:
I know it was already defined this way, but I think `undefined` is redundant here...